src/app/teams/teams-view.component.ts

```diff
@@ -300,4 +351,12 @@ export class TeamsViewComponent implements OnInit, OnDestroy {
     ).subscribe(() => this.planetMessageService.showMessage(`${member.name} has been made Leader`));
   }
 
+  removeCourse(item) {
+    const index = this.team.courses.indexOf(item);
+    this.team.courses.splice(index, 1);
+    this.couchService.updateDocument('teams/', this.team).subscribe(() => {
```

Comment:
If you use `this.teamsService.updateTeam(this.team).subscribe((newTeam) => {` you can then set `this.team` to `newTeam` to update its `_rev` and allow further changes to the team.